src/adapters/collateralized_simple_interest_loan_adapter.ts

```diff
@@ -296,6 +325,49 @@ export class CollateralizedSimpleInterestLoanAdapter implements Adapter.Interfac
         ).toArray();
     }
 
+    /**
+     * Seizes the collateral from the given debt agreement and
+     * transfers it to the debt agreement's beneficiary.
+     *
+     * @param {string} agreementId
+     * @returns {Promise<string>} The transaction's hash.
+     */
+    public async seizeCollateral(agreementId: string): Promise<string> {
+        this.assert.schema.bytes32("agreementId", agreementId);
+
+        await this.assertCollateralSeizeable(agreementId);
+
+        const collateralizerContract = await this.contractsAPI.loadCollateralizerAsync();
+
+        return collateralizerContract.seizeCollateral.sendTransactionAsync(
+            agreementId,
+            // FIXME: For default "from", was getting "invalid address" and default gas reverts.
+            { from: ACCOUNTS[1].address, gas: 4712388 },
```

Comment:
For the time being let's choose more reasonable gas limits.  Though [this PT task](https://www.pivotaltracker.com/story/show/156644350) will at some point obviate the need for a hard-coded limit, there's a scenario in which this code will get pushed out to public before then.  In that case, choosing the absolute-maximum gas amount by default will give users really scary looking fee amounts when they need to confirm transactions.

We can still choose a liberal maximum here -- but it can be at least an order of magnitude smaller.

Reply:
Also, `ACCOUNTS[1]` isn't available outside of a test context, so we need to find a better solution here.  Perhaps the `TransactionOptions` utility can be of use here.